src/main/frontend/views/$layout.tsx

```diff
@@ -0,0 +1,41 @@
+import { AppLayout } from '@vaadin/react-components/AppLayout.js';
+import { DrawerToggle } from '@vaadin/react-components/DrawerToggle.js';
+import {createMenuItems, useViewConfig} from '@vaadin/hilla-file-router/runtime.js';
+import { ProgressBar } from '@vaadin/react-components/ProgressBar';
+import { Suspense, useEffect } from 'react';
+import { NavLink, Outlet } from 'react-router-dom';
+
+const navLinkClasses = ({ isActive }: any) => {
+  return `block rounded-m p-s ${isActive ? 'bg-primary-10 text-primary' : 'text-body'}`;
+};
+
+export default function MainLayout() {
+  const currentTitle = useViewConfig()?.title ?? 'Hybrid example';
```

Comment:
What about reading the default title from HTML?



```suggestion

const defaultTitle = document.title;

export default function MainLayout() {
  const currentTitle = useViewConfig()?.title ?? defaultTitle;
```